cpp/src/arima/batched_arima.cu

```diff
@@ -252,17 +248,244 @@ void batched_loglike(cumlHandle& handle, double* d_y, int num_batches, int nobs,
 
     batched_kalman_filter(handle, y_diff, nobs - d, d_Tar, d_Tma, p, q,
                           num_batches, loglike, d_vs);
+
     allocator->deallocate(y_diff, sizeof(double) * num_batches * (nobs - 1),
                           stream);
   } else {
     throw std::runtime_error("Not supported difference parameter: d=0, 1");
   }
+  allocator->deallocate(d_Tar, sizeof(double) * p * num_batches, stream);
+  allocator->deallocate(d_Tma, sizeof(double) * q * num_batches, stream);
+  ML::POP_RANGE();
+}
+
+void batched_loglike(cumlHandle& handle, double* d_y, int num_batches, int nobs,
+                     int p, int d, int q, double* d_params,
+                     std::vector<double>& loglike, double* d_vs, bool trans) {
+  ML::PUSH_RANGE(__FUNCTION__);
```

Comment:
Not too sure about this. IIRC, `__FUNCTION__` is non-standard. Maybe better to use `__func__` which seems to be now part of c++11?

Reply:
```suggestion
  ML::PUSH_RANGE(__func__);
```

Reply:
Sure, it's actually simply copied/pasted from the other overload of this function (I added one overload with a different interface), but I can make the change for all the functions in these files that are using `__FUNCTION__`